.github/workflows/publish-hotfix-release.yml

```diff
@@ -0,0 +1,267 @@
+                   { dir: edc-controlplane, img: edc-controlplane-postgresql-azure-vault },
+                   { dir: edc-dataplane,    img: edc-dataplane-azure-vault },
+                   { dir: edc-dataplane,    img: edc-dataplane-hashicorp-vault },
+                   { dir: edc-tests/runtime, img: mock-connector }]
+    steps:
+      - uses: actions/checkout@v4
+      - name: Export RELEASE_VERSION env
+        run: |
+          echo "RELEASE_VERSION=${{ needs.release-version.outputs.RELEASE_VERSION }}" >> $GITHUB_ENV
+      - uses: ./.github/actions/publish-docker-image
+        name: Publish ${{ matrix.variant.img }}
+        with:
+          docker_tag: ${{ env.RELEASE_VERSION }}
+          rootDir: ${{ matrix.variant.dir }}/${{ matrix.variant.img }}
+          imagename: ${{ matrix.variant.img }}
+          docker_user: ${{ secrets.DOCKER_HUB_USER }}
+          docker_token: ${{ secrets.DOCKER_HUB_TOKEN }}
+          do_push: 'true'
+
+  # Release: Helm Charts
+  helm-release:
+    name: Publish new helm release
+    needs: [ prepare-hotfix-release, release-version ]
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+      packages: write
+      pages: write
+
+    if: needs.release-version.outputs.RELEASE_VERSION
+    steps:
+      - name: Export RELEASE_VERSION env
+        run: |
+          echo "RELEASE_VERSION=${{ needs.release-version.outputs.RELEASE_VERSION }}" >> $GITHUB_ENV
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+      - name: Install Helm
+        uses: azure/setup-helm@v4
+        with:
+          version: v3.8.1
+      - name: Package helm, update index.yaml and push to gh-pages
+        run: |
+          # Prepare git env
+          git config user.name "eclipse-tractusx-bot"
+          git config user.email "tractusx-bot@eclipse.org"
+
+          # Package all charts
+          find charts -name Chart.yaml -not -path "./edc-tests/*" | xargs -n1 dirname | xargs -n1 helm package -u -d helm-charts
+
+          git checkout gh-pages || git checkout -b gh-pages
+          git pull --rebase origin gh-pages
+
+          # Generate helm repo index.yaml
+          helm repo index . --merge index.yaml --url https://${GITHUB_REPOSITORY_OWNER}.github.io/${GITHUB_REPOSITORY#*/}/
+
+          # Commit and push to gh-pages
+          git add index.yaml helm-charts
+          git commit -s -m "Release ${{ env.RELEASE_VERSION }}"
+
+          git push origin gh-pages
+
+  # Release: GitHub tag & release; Merges back releases into main; Starts a new development cycle;
+  github-release:
+    name: Publish new github release
+    needs: [ prepare-hotfix-release, release-version, maven-release, docker-release, helm-release ]
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+      packages: write
+      pages: write
+      pull-requests: write
+    if: needs.release-version.outputs.RELEASE_VERSION
```

Comment:
?? if `RELEASE_VERSION` contains `${{ inputs.version }}`, which is mandatory, then this expression is always `true` (and thus spurious)

Reply:
[c240841](https://github.com/eclipse-tractusx/tractusx-edc/pull/1511/commits/c24084146ae9dad6a139561ae477c8eef49b2764)